server/lib/routes/post-csp.js

```diff
@@ -0,0 +1,62 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * Route to report CSP Violations to metrics
+ */
+
+var DEFAULT_RATE_LIMIT = 90;
```

Comment:
This constant seems like it belongs in the [lib/configuration.js csp](https://github.com/mozilla/fxa-content-server/blob/0a6fa427ab2f721ebb327dd91c0aac389219fabe/server/lib/configuration.js#L59) section.